build warnings and dead code

While rebuilding BruTile (to create pdb files), I received 13 compiler warnings.
`warning CS0472: The result of the expression is always true` seems to be dead code from the conversion of 'level' from string to int.
And the others are about deprecated build-task elements.

--- a/BruTile/Web/BasicRequest.cs
+++ b/BruTile/Web/BasicRequest.cs
@@ -121,10 +121,10 @@ namespace BruTile.Web
         /// </summary>
         /// <param name="tileX">Tile X coordinate.</param>
         /// <param name="tileY">Tile Y coordinate.</param>
-        /// <param name="levelId">Level of detail, from 1 (lowest detail)
+        /// <param name="level">Level of detail, from 1 (lowest detail)
         /// to 23 (highest detail).</param>
         /// <returns>A string containing the QuadKey.</returns>
-        /// Stole this methode from this nice blog: http://www.silverlightshow.net/items/Virtual-earth-deep-zooming.aspx. PDD.
+        /// Stole this method from this nice blog: http://www.silverlightshow.net/items/Virtual-earth-deep-zooming.aspx. PDD.
         private static string TileXyToQuadKey(int tileX, int tileY, int level)
         {
             var quadKey = new StringBuilder();

--- a/Samples/BruTile.Samples.Common/RenderSelect.cs
+++ b/Samples/BruTile.Samples.Common/RenderSelect.cs
@@ -44,7 +44,7 @@ namespace BruTile.Samples.Common
             
                 if (tile == null)
                 {
-                    if (nextLevelId != null) SelectRecursive(selection, cache, schema, tileInfo.Extent.Intersect(extent), nextLevelId);
+                    SelectRecursive(selection, cache, schema, tileInfo.Extent.Intersect(extent), nextLevelId);
                 }
                 else
                 {
@@ -53,7 +53,7 @@ namespace BruTile.Samples.Common
                     // semi transparent one.
                     if (IsSemiTransparent(tile))
                     {
-                        if (nextLevelId != null) SelectRecursive(selection, cache, schema, tileInfo.Extent.Intersect(extent), nextLevelId);
+                        SelectRecursive(selection, cache, schema, tileInfo.Extent.Intersect(extent), nextLevelId);
                     }
                 }
             }

